c2corg_common/attributes.py

```diff
@@ -644,3 +645,85 @@
     'tourism',
     'magazine'
 ]
+
+event_types = [
+    'avalanche',
+    'stone_fall',
+    'falling_ice',
+    'other_fall',
+    'crevasse_fall',
+    'roped_fall',
+    'physical_failure',
+    'lightning',
+    'other'
+]
+
+author_statuses = [
+    'primary_impacted',
+    'secondary_impacted',
+    'internal_witness',
+    'external_witness'
+]
+
+activity_rates = [
+    'activity_rate_150',
+    'activity_rate_50',
+    'activity_rate_30',
+    'activity_rate_20',
+    'activity_rate_10',
+    'activity_rate_5',
+    'activity_rate_1'
+]
+
+nb_outings = [
+    'nb_outings_4',
+    'nb_outings_9',
+    'nb_outings_14',
+    'nb_outings_15',
+]
+
+genders = [
+    'male',
+    'female'
+]
+
+previous_injuries = [
+    'no',
+    'previous_injuries_2',
+    'previous_injuries_3'
+]
+
+severities = [
+    'no',
+    '1 to 3 days',
```

Comment:
I see that ``public_transportation_ratings`` values have blank spaces
https://github.com/c2corg/v6_common/blob/a390109bd218e1d3505aae7edc19baa920f6aa83/c2corg_common/attributes.py#L107-L113
and I have not noticed problems with those values and their translations.

So perhalps blank spaces are OK. On the other hand I am sure we had problems with blank spaces for some attributes and had to use underscores instead. I can't remember the problem. Can anyone remind about this?